.mypy.ini

```diff
@@ -0,0 +1,95 @@
+[mypy]
+python_version = 3.7
```

Comment:
Can't we use a newer version?

Reply:
That's the current lowest version of python that manim actively supports last time I checked.

Reply:
So we should use the lowest possible version? Why so, if I may ask?

Reply:
I'm not 100% sure what that setting does, but possibly breaking support for versions we still claim we support tends to be a bad thing.

Reply:
Ah, I don't remember why it was added, but it's optional and does not really affect the end result so I removed it.